src/schemathesis/models.py

```diff
@@ -224,6 +227,11 @@ def call_wsgi(self, app: Any = None, headers: Optional[Dict[str, str]] = None, *
                 "Please, set `app` argument in the schema constructor or pass it to `call_wsgi`"
             )
         data = self.as_werkzeug_kwargs(headers)
+        data["headers"] = data["headers"] or {}
+        if headers is not None:
+            data["headers"].update(headers)
```

Comment:
These lines are not needed, since headers merging is handled in the `as_werkzeug_kwargs` method.

Reply:
Yes, it  true, I will remove these lines.